litellm/utils.py

```diff
@@ -25,6 +25,7 @@
     dataclass,
     field,
 )  # for storing API inputs, outputs, and metadata
+
```

Comment:
it looks like there are a large amount of changes to this file. Can we avoid this ?

Reply:
and it looks like they're all formatting changes

Reply:
VSCode auto-format on save 😅 
Done @ishaan-jaff